shell/components/nav/__tests__/Type.test.ts

```diff
@@ -1,167 +1,365 @@
-import { mount, RouterLinkStub } from '@vue/test-utils';
+import { shallowMount, RouterLinkStub, createLocalVue } from '@vue/test-utils';
 import Type from '@shell/components/nav/Type.vue';
+import { createChildRenderingRouterLinkStub } from '@shell/utils/unit-tests/ChildRenderingRouterLinkStub';
+import { TYPE_MODES } from '@shell/store/type-map';
+
+// Configuration
+const activeClass = 'router-link-active';
+const exactActiveClass = 'router-link-exact-active';
+const rootClass = 'root';
+const favoriteClass = 'favorite';
+const defaultCount = 1;
+const defaultRouteTypeProp = {
+  name:  'route-type',
+  route: 'route',
+  exact: true,
+  mode:  TYPE_MODES.FAVORITE
+};
+const defaultLinkTypeProp = {
+  link:   'link-type-link',
+  label:  'link-type-label',
+  target: 'link-type-target'
+};
 
 // Mandatory to mock vue-router in this test
 jest.mock('vue-router');
+const storeMock = {
+  getters: {
+    currentStore:    () => 'cluster',
+    'cluster/count': () => defaultCount,
+  }
+};
+const routerMock = { resolve: () => ({ route: { path: 'whatever' } }) };
+const localVue = createLocalVue();
 
-// Configuration text
-const className = 'router-link-active';
+localVue.directive('cleanHtml', (identity) => identity);
```

Comment:
Is this not already mocked in the setup? With this extra?

Reply:
You are right, it is actually missing in the [global Jest setup](https://github.com/rancher/dashboard/blob/master/jest.setup.js). I got confused with the branch I was working on. I will double check next time.